compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Monoid.hs

```diff
@@ -136,6 +154,11 @@ renderFolder formatter fileMap =
                 [ SameFile <$ guard (Set.member anchor localAnchors)
                 , SameFolder <$> Map.lookup anchor globalAnchors
                 ]
+            lookupReference ref = asum
+                [ lookupAnchor (referenceAnchor ref)
+                , External <$> (packageURI =<< referencePackage ref)
+                ]
```

Comment:
looks like you can factor this function out and reuse it.

Reply:
Thanks! I'm refactoring it :-)